Dockerfiles/oc-dev-docs/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG OC_DEV_DOCS_BASE=ubuntu:22.04
+FROM $OC_DEV_DOCS_BASE
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
```

Comment:
Maybe replace with simply apt?

Reply:
Why?
1) apt is for end-users, while apt-get is for backends (this is a backend).
2) The existing CI code uses apt-get, not apt.

Reply:
I'm not a Linux expert nor I wanna judge your choices, but I wanna point out that:

> apt is for end-users, while apt-get is for backends (this is a backend).

Your statement isn't wrong but unless you're using old low-level commands (e.g. `apt-cache`, `apt-config` etc) you won't notice any practical difference between `apt` and `apt-get`.

> The existing CI code uses apt-get, not apt.

From what I saw in the existing CI code, there are only `update` and `install` comands in:

- [`analyze.yml/analyze-docs-linux`](https://github.com/acidanthera/OpenCorePkg/blob/6f25ba4f2c60d097f0ae54e8ecb765c6faa5e2db/.github/workflows/analyze.yml#L95-L96)
- [`analyze.yml/analyze-coverity`](https://github.com/acidanthera/OpenCorePkg/blob/6f25ba4f2c60d097f0ae54e8ecb765c6faa5e2db/.github/workflows/analyze.yml#L113-L114)
- [`build.yml/build-linux-clangpdb-gcc5`](https://github.com/acidanthera/OpenCorePkg/blob/6f25ba4f2c60d097f0ae54e8ecb765c6faa5e2db/.github/workflows/build.yml#L70-L71)
- [`build.yml/build-linux-clangdwarf`](https://github.com/acidanthera/OpenCorePkg/blob/6f25ba4f2c60d097f0ae54e8ecb765c6faa5e2db/.github/workflows/build.yml#L100-L101)
- [`uncrustify.yml/analyze-uncrustify`](https://github.com/acidanthera/OpenCorePkg/blob/6f25ba4f2c60d097f0ae54e8ecb765c6faa5e2db/.github/workflows/uncrustify.yml#L22-L23)

Therefore there shouldn't be any issue in switching from `apt-get` to `apt` unless you plan to use some old low-level commands as mentioned before.

P.S. I'd like to share this article which may better explain the changes between `apt` and `apt-get`: https://itsfoss.com/apt-vs-apt-get-difference/

Reply:
Why would we use a tool that is explicitly not built for the scenario we are in? apt does not even have some kind of stable output, unlike apt-get, which is sometimes parsed by scripts. Yes, we do not parse here, but there is neither an issue with apt-get, nor any advantage with apt here.